spec/models/package_user_spec.rb

```diff
@@ -1,3 +1,19 @@
+# == Schema Information
+#
+# Table name: package_user
+#
+#  id         :integer          not null, primary key
+#  package_id :integer
+#  user_id    :integer
+#  created_at :datetime
+#  updated_at :datetime
+#  active     :boolean          default(TRUE), not null
+#
+# Indexes
+#
+#  index_package_user_on_package_id_and_user_id  (package_id,user_id) UNIQUE
+#
```

Comment:
We usually don't include these headers because of the risk of them becoming outdated.  I can live with them in the models but I think it's overkill to have them in specs too, too much duplication imo